README.md

```diff
@@ -65,7 +65,7 @@ with `.erb.` appended. The templates are
 rendered using values from a file called `config_defaults.yml` in the root (not
 moduleroot) of the configuration directory. The default values can be
 overridden or extended by adding a file called `.sync.yml` to the module itself.
-This allows us to, for example, have a set of "required" gems that are added
+This allows us to, for example, to have a set of "required" gems that are added
```

Comment:
This extra `to` shouldn't be added.

Reply:
Great you spotted that! Do you want a separate PR for it?

Reply:
Oh, I understand. Isn't the place of the "to" unusual? That's why it irritated me.